### PR TITLE
Hide disabled question

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3644

image

image

Meet the 17.75s! They're a family of critics. Under is pretty harsh, but Over is excited about everything.

### CODE REVIEW

**Review by jbalboni — APPROVED** [review]
LGTM. I think you can take a stab at the DefinitionTester changes in here or make a separate issue for them.

**Review by cehsu — APPROVED** [review]

File: src/js/pensions/config/form.js [hunk 1]
```diff
@@ -96,6 +96,11 @@ function isBetween18And23(childDOB) {
   return moment(childDOB).isBetween(moment().startOf('day').subtract(23, 'years'), moment().startOf('day').subtract(18, 'years'));
 }
 
+// Actually checks if they're under 17.75, but that doesn't make for a good function name
+function isUnder18(childDOB) {
```
Comment (cehsu) [inline src/js/pensions/config/form.js]: Could maybe go with a non-numerical descriptor such as 'isEligibleForDisabilitySupport' and then provide comments explaining how this is defined?

Comment (jbalboni) [inline src/js/pensions/config/form.js]: ^ That's a good suggestion

Comment (cvalarida) [inline src/js/pensions/config/form.js]: Cool. My other thought was `isUnderSeventeenYearsAndNineMonthsOld` or even `hasLivedThroughRoughlySeventyOneChangesOfTheSeasons`, but...I think I'll go with your suggestion. 👍

**Comment by cvalarida** [comment]
`DefinitionTester` ticket: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3690

**Pull request merged**